<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recieveInventoryWidget</class>
 <widget class="QWidget" name="recieveInventoryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recieve Inventory</string>
  </property>
  <widget class="QPushButton" name="recieveInventorySaveButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>Recieve Inventory</string>
   </property>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>411</width>
      <height>182</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="1">
      <widget class="QComboBox" name="recieveInvItemID"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Item Name</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Item Description</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="recieveInvDescription">
       <property name="placeholderText">
        <string>Auto-filled from Item ID</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Item Cost</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="recieveInvCost">
       <property name="placeholderText">
        <string>Auto-filled from Item ID</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Quantity Received</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QSpinBox" name="recieveInvQty"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="recieveInvItemName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Internal Part ID</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="recieveInvInternalPartID">
       <property name="placeholderText">
        <string>Internal Part ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Item ID</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="recieveInventoryNoteBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>441</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Notes:</string>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Use this module while receiving new inventory.</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
